agent/replacecall/src/main/resources/default-replacements.txt

```diff
@@ -7,9 +7,8 @@ java.lang randoop.mock.java.lang
 java.awt randoop.mock.java.awt
 javax.swing randoop.mock.javax.swing
 
-// Replacements to instance method calls of java.util.Random.
-java.util.Random randoop.mock.java.util.MockRandom
-// Currently, the replacecalls agent cannot replace constructors.
-// This can be reinstated, and the replacements for all the methods of
-// Random removed, when replacecalls can handle constructors.
-// java.util.Random.<init>() randoop.mock.java.util.RandomReplace.randomWithSeedZero()
+// Replacements for java.util.Random.
+// Make Random deterministic by always using a random seed of 0.
+java.util.Random.<init>() randoop.mock.java.util.RandomReplace.randomWithSeedZero()
+java.util.Random.<init>(long) randoop.mock.java.util.RandomReplace.randomWithSeedZero(long)
```

Comment:
I suppose you could leave the methods defined and these two lines commented out, with a disclaimer about changing program behavior.  But I wonder whether they are needed at all.